src/components/ScheduleDetail/ScheduleInfoList/ScheduleInfoList.component.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { ScheduleStatus, ValueOf } from '@/types';
+
+import * as Styled from './ScheduleInfoList.styled';
+import { formatDate, getScheduleStatusText } from '@/utils';
+
+interface ScheduleInfoListProps {
+  generationNumber: number;
+  name: string;
+  createdAt: string;
+  startedAt: string;
+  publishedAt?: string;
+  status: ValueOf<typeof ScheduleStatus>;
+}
+
+const ScheduleInfoList = ({
+  name,
+  generationNumber,
+  startedAt,
+  createdAt,
+  publishedAt,
+  status,
+}: ScheduleInfoListProps) => {
+  return (
+    <Styled.ScheduleInfoList>
+      <li>
+        <Styled.ScheduleInfoLabel>스케줄 명</Styled.ScheduleInfoLabel>
+        <Styled.ScheduleInfoValue>{name}</Styled.ScheduleInfoValue>
+      </li>
+      <li>
+        <Styled.ScheduleInfoLabel>기수</Styled.ScheduleInfoLabel>
+        <Styled.ScheduleInfoValue>{generationNumber}기</Styled.ScheduleInfoValue>
+      </li>
+      <li>
+        <Styled.ScheduleInfoLabel>스케줄 일시</Styled.ScheduleInfoLabel>
+        <Styled.ScheduleInfoValue>
+          {formatDate(startedAt, 'YYYY년 M월 D일 A hh시 mm분')}
+        </Styled.ScheduleInfoValue>
+      </li>
+      <li>
+        <Styled.ScheduleInfoLabel>등록 일시</Styled.ScheduleInfoLabel>
+        <Styled.ScheduleInfoValue>
+          {formatDate(createdAt, 'YYYY년 M월 D일 A hh시 mm분')}
+        </Styled.ScheduleInfoValue>
+      </li>
+      <li>
+        <Styled.ScheduleInfoLabel>배포 일시</Styled.ScheduleInfoLabel>
+        <Styled.ScheduleInfoValue>
+          {formatDate(publishedAt, 'YYYY년 M월 D일 A hh시 mm분')}
+        </Styled.ScheduleInfoValue>
+      </li>
+      <li>
+        <Styled.ScheduleInfoLabel>배포 상태</Styled.ScheduleInfoLabel>
+        <Styled.ScheduleInfoValue>{getScheduleStatusText(status)}</Styled.ScheduleInfoValue>
+      </li>
+    </Styled.ScheduleInfoList>
+  );
```

Comment:
```suggestion
  const ScheduleInfolistItem = useMemo(
    () => ({
      '스케줄 명': name,
      기수: generationNumber,
      '스케줄 일시': formatDate(startedAt, 'YYYY년 M월 D일 A hh시 mm분'),
      '등록 일시': formatDate(createdAt, 'YYYY년 M월 D일 A hh시 mm분'),
      '배포 일시': formatDate(publishedAt, 'YYYY년 M월 D일 A hh시 mm분'),
      '배포 상태': getScheduleStatusText(status),
    }),
    [name, generationNumber, startedAt, createdAt, publishedAt, status],
  );

  return (
    <Styled.ScheduleInfoList>
      {Object.entries(ScheduleInfolistItem).map(([infoName, infoValue]) => (
        <li>
          <Styled.ScheduleInfoLabel>{infoName}</Styled.ScheduleInfoLabel>
          <Styled.ScheduleInfoValue>{infoValue}</Styled.ScheduleInfoValue>
        </li>
      ))}
    </Styled.ScheduleInfoList>
  );
```

꼭 이런 식은 아니더라도 메모와 map을 활용하면 어떨까요?

Reply:
@sanoopark  상수가 조금 길어지긴 해도 요렇게 하는게 map 돌리느데서 조금 더 가독성이 좋은거 같아 요렇게 했어요

https://github.com/mash-up-kr/mash-up-admin-fe/pull/247/commits/d89d9157c044c9492fd5868d45999016d130bac1